### PR TITLE
Adds Helm installer

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/1253

### CODE REVIEW

File: service/deployer/deployer.go [hunk 1]
```diff
@@ -115,22 +119,16 @@ func (s *standardDeployer) Boot() {
 	}
 
 	for deploymentEvent := range deploymentEventChannel {
-		s.logger.Log("debug", "installing chart", "name", deploymentEvent.Name)
```
Comment (JosephSalisbury) [inline service/deployer/deployer.go]: We handle this logging, and the following two log lines, inside the `Installer` now.

File: service/deployer/installer/helm/helm.go [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	}
+	defer os.Remove(tarballPath)
+
+	i.logger.Log("debug", "downloaded chart", "tarball", tarballPath)
+
+	if err := i.runHelmCommand(
+		"install",
+		"install",
+		tarballPath,
+		// "--values",
```
Comment (JosephSalisbury) [inline service/deployer/installer/helm/helm.go]: This is me hacking to make the installation work without dealing with configuration management yet.

**Comment by JosephSalisbury** [comment]
Cool. I have successfully installed the api to minikube with this PR (and a hardcoded values file).

I'm gonna cut this here, and introduce a `Configurer` interface/service next, to properly handle the configuration management, which we'll use with the helm installer to provide the values file.

**Comment by JosephSalisbury** [comment]
@xh3b4sd come at me bro

File: service/deployer/installer/helm/helm.go (hunk 2 above)
Comment (xh3b4sd) [inline service/deployer/installer/helm/helm.go]: Not sure if you want to merge it with this here.

Comment (JosephSalisbury) [inline service/deployer/installer/helm/helm.go]: I'm changing it in the next PR, it's just so I don't forget where I need to go :D

**Pull request merged**